onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -87,24 +87,26 @@ void BasicBackend::StartAsyncInference(Ort::CustomOpApi& ort,
     auto graph_input_buffer = graph_input_blob->buffer()
                                   .as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type*>();
     size_t input_data_size = graph_input_blob->byteSize();
-
-    auto tensor_shape = ort.GetTensorTypeAndShape(input_tensors[i]);
+    
+    const OrtValue* tensor = ort.KernelContext_GetInput(context, subgraph_context_.input_names.at(input_info_iter->first));
```

Comment:
If we are using input_info_iter->first 4 times, it's better to have a variable for that right? It makes the code look cleaner.